packages/components/src/box-control/styles/box-control-styles.ts

```diff
@@ -1,80 +1,40 @@
 /**
  * External dependencies
  */
-import { css } from '@emotion/react';
 import styled from '@emotion/styled';
 /**
  * Internal dependencies
  */
-import { Flex } from '../../flex';
+import BoxControlIcon from '../icon';
+import Button from '../../button';
+import { HStack } from '../../h-stack';
+import RangeControl from '../../range-control';
 import BaseUnitControl from '../../unit-control';
-import { rtl } from '../../utils';
-import type { BoxUnitControlProps } from '../types';
+import { space } from '../../utils/space';
 
-export const Root = styled.div`
-	box-sizing: border-box;
-	max-width: 235px;
-	padding-bottom: 12px;
-	width: 100%;
+export const UnitControl = styled( BaseUnitControl )`
```

Comment:
Nit: For consistency with the other styled components.

```suggestion
export const StyledUnitControl = styled( UnitControl )`
```